elements/pfe-avatar/src/pfe-avatar.js

```diff
@@ -0,0 +1,352 @@
+/*
```

Comment:
@mwcz You can delete the copyright from this file now.

Reply:
:+1: